.changeset/olive-bears-act.md

```diff
@@ -0,0 +1,107 @@
+---
+'@primer/react': major
+---
+
+![image](https://user-images.githubusercontent.com/13340707/154948443-60d70cc7-4f26-444f-8366-7963336be53c.png)
```

Comment:
Can we update the caption here to say something like `FormControl component using a label, text input, validation and hint`?